src/main/resources/META-INF/resources/primefaces-extensions/codescanner/codescanner-widget.js

```diff
@@ -36,12 +45,7 @@ PrimeFaces.widget.ExtCodeScanner = PrimeFaces.widget.BaseWidget.extend({
                 };
                 $this.callBehavior('codeScanned', ext);
             }
-
         }
-
-        var $this = this;
-        this.video = $('video', this.jq)[0];
-        this.codeReader = getReader(cfg.type);
         this.codeReader.listVideoInputDevices()
                 .then((videoInputDevices) => {
                     var deviceId = videoInputDevices[0].deviceId;
```

Comment:
Do not use lambdas.  PF supports ES5 only.

Reply:
Ah.. Zxing is ES6.. so we cannot use that?

Reply:
.. I cannot find any info on what version PF is using... Isn't mentioning the minimum ES version in the documentation good enough in order to be able to use the library?

Reply:
Well I just know from experience and previous commits. can you try and run `mvn clean package -Pcompress` and see if it fails